DataConnectors/Agari/AzureFunctionAgari/run.ps1

```diff
@@ -232,12 +273,19 @@ $TimeStampField = "DateValue"
 
 #Check if BP is enabled and we got a token
 if (($bpEnabled) -and ($bpToken)) {
+    #Check to see if start time empty - usally first run, otherwise set the time from the env variable
+    if ($BPlastLog){
+        $startdate = $BPlastLog.substring(1)  
+    } else {
+        $startdate=(Get-Date (get-date).addMinutes(-6) -UFormat "+%Y-%m-%dT%H:%M:00:001Z")
```

Comment:
Why are we subtracting 6 minutes ?

Reply:
To get a full 5 minutes (i.e startdate is 6 mins in the past and the enddate is 1 min in the past. I did this to account for any potential delays in the timer script.